plugins/main/scripts/tag.py

```diff
@@ -17,7 +17,7 @@
 # Wazuh version: major.minor.patch
 version = '4.6.0'
 # App's revision number (previous rev + 1)
-revision = '01'
+revision = '02'
```

Comment:
**nitpick**: undone the change

Reply:
Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.